resume/src/app/app.component.html

```diff
@@ -0,0 +1,18 @@
+<body class="bg-gray-light antialiased">
+<main class="p-5 max-w-5xl ml-auto mr-auto">
```

Comment:
I really like the setup here, I think this is a nice example for a lot of a lot of different functionalities in Tailwind (like the colon in g:grid-cols-3 and a lot of styling). Maybe we can use this as demo